app/src/main/java/com/teamagam/gimelgimel/app/map/esri/IgnoreDragMapOnTouchListener.java

```diff
@@ -0,0 +1,20 @@
+package com.teamagam.gimelgimel.app.map.esri;
+
+import android.view.MotionEvent;
+import com.esri.android.map.MapOnTouchListener;
+
+class IgnoreDragMapOnTouchListener extends MapOnTouchListener {
+  IgnoreDragMapOnTouchListener(EsriGGMapView mapView) {
```

Comment:
Well, I thought about it and since I'm actually using the context coming from the view I decided to make this class' users life easier by writing it this way.
I couldn't see any benefit to the alternative.